src/utils/saml.ts

```diff
@@ -1175,6 +1180,15 @@ export const preValidateResponse: PreValidateResponseT = (
         )
       )
         .chain(Attributes => {
+          if (
+            !strictValidationOptions ||
```

Comment:
is there any way to extract this condition at the beginning of this method, something like
`const hasStrictValidation = strictValidationOptions && strictValidationOptions[idpIssuer]`
(at the cost of reading issuer from the cache twice)

Reply:
At the very top of the method is not possible, because to read the Request cache is needed the `InResponseTo` value inside the Response. What do you suggest?

Reply:
since we check that the request issuer equals the response one,
can't we get the issuer from the response instead ? 

ie.

`const hasStrictValidation = getSamlIssuer(doc).map(issuer => strictValidationOptions[idpIssuer]).getOrElse(false)
`

Reply:
Good suggestion :) 
Fixed in 0c75fc8